src/common/components/hosts/HostsTable.css

```diff
@@ -12,8 +12,4 @@
 
 .hosts-table {
   height: 1rem; /* Workaround for: https://bugzilla.redhat.com/show_bug.cgi?id=1879378 in FF; The actual table height will be recalculated accordingly. */
-}
-
-table.hosts-table.pf-m-compact {
-  height: auto;
 }
```

Comment:
Please, add a newline :slightly_smiling_face: 
```suggestion
}

```

Reply:
Maybe we could we making this rule into Prettier?